### DISCUSSION TITLE
Add chart to the banner

### DISCUSSION THREAD

**Discussion by Kent-Romania** [tltneon/lgsl#88] [General]
Hello. can you help me add charts to the banner?
it's about userbar.

**Comment by tltneon** [comment]
Hello @Kent-Romania 
That's a nice idea! I could add it onto wide banner

**Reply by TacTicTow** [reply]
I was actually about to request something like this. I have already edited the way my banners look because I just really dont like static banners. Need something smaller that is more in line with what you see on gametacker site. Something like this would be great. 
![chart](https://user-images.githubusercontent.com/97808072/153698308-116eb0af-02a7-4582-895c-53131adadae2.jpg)

For my particular banner, 130x50 is an ideal size.

**Comment by Kent-Romania** [comment]
**I am waiting for an update, this banner charts would be useful to me, I hope it will be soon. Thanks: X.**

**Reply by TacTicTow** [reply]
He already implemented it. Download the latest and you are good to go. Using them in our banner now.

https://saltybawls.com/lgsl/userbar.php?s=21&t=2

**Comment by TacTicTow** [comment]
He has already implemented it. Download the current and youre good to go.  :)

https://saltybawls.com/lgsl/userbar.php?s=21&t=2

**Reply by Kent-Romania** [reply]
> 
**Hello. I downloaded the latest version of the version but I have no change, can you leave me the download link here? Maybe I'm omitting something. Thanks**

**Comment by TacTicTow** [comment]
https://github.com/tltneon/lgsl/archive/refs/heads/master.zip

If you look at the list of files

https://github.com/tltneon/lgsl

The userbars.php and charts.php have been changed. Those are really the only files you need.

**Reply by Kent-Romania** [reply]
**Thanks**

**Comment by Kent-Romania** [comment]
**TacTicTow** sorry for the inconvenience but I would like my lgsl flags to look like this: https://saltybawls.com/lgsl/userbar.php?s=21&t=2
can you help me? I really like changing the image of the banner with each game mode, and as they are placed in the banner mentioned above, it would be useful for me to have a similar lgsl: X

**Reply by TacTicTow** [reply]
Yea I prefer to not have a static banner either which is why I modified the code myself. That banner is my own thing. What is your email address? I will send you my files and fonts. Did you want my backgrounds as well or will you make your own?

**Reply by Kent-Romania** [reply]
mail: dragneliviumarian@yahoo.com
steam: kentliviu94
I want it to be exactly as you have the lgslul benerele format. they are beautiful, I like what the flags look like. if you help me I will be grateful: x
